packages/ui/textEditor/Lexical/index.jsx

```diff
@@ -159,6 +162,32 @@ export function Editor(props) {
           onBlur(event);
         }
       }, LowPriority),
+      editor.registerCommand(
+        PASTE_COMMAND,
+        (event) => {
+          const pastedText = event.clipboardData?.getData("text/html");
+          let processedText = pastedText;
+
+          if (pastedText?.includes("&lt;")) {
+            const doc = new DOMParser().parseFromString(pastedText, "text/html");
+            processedText = doc.body.textContent;
+          }
+
+          if (pastedText?.includes("text-decoration:underline line-through;")) {
+            processedText = parsePastedLexicalContent(processedText);
+          }
+
+          if (processedText !== pastedText) {
+            const pasteEvent = createPasteEventWithData(processedText);
+            editor.dispatchCommand(PASTE_COMMAND, pasteEvent);
+
+            return true;
+          }
+
+          return false;
+        },
+        HighPriority,
+      ),
```

Comment:
@AlexBabusanu  the implementation looks good. For uniformity, could you move this into a command, like we have, for example, for click ?
You can look for examples in the `Toolbar/commands.jsx` file.